src/ontology/fbbt-edit.obo

```diff
@@ -156969,6 +157050,10 @@ is_a: FBbt:00007060 ! multi-cell-component structure
 relationship: bearer_of PATO:0060002 ! male-specific
 relationship: part_of FBbt:00005320 ! individualization stage spermatid
 
+[Term]
+id: http://flybase.org/reports/FBgn0034935 ! Orcokinin
+is_a: SO:0000704
```

Comment:
I think this has appeared because you added http://flybase.org/reports/FBgn0034935 under SO:0000704(?) 
Terms to be imported should be added directly under owl:Thing so that they don't appear in the editors' file (if you assert a superclass in the -edit file, that has to be stored in the -edit file). I will clarify this in the readme.
You can delete these four lines.